web/src/lib/cms/previews/TaskPreview.tsx

```diff
@@ -1,8 +1,9 @@
-import { TaskElement } from "@/components/TaskElement";
 import { PreviewProps } from "@/lib/cms/helpers/previewHelpers";
 import { usePageData } from "@/lib/cms/helpers/usePageData";
 import { usePreviewRef } from "@/lib/cms/helpers/usePreviewRef";
-import { Task } from "@/lib/types/types";
+import { createEmptyTaskDisplayContent, Task } from "@/lib/types/types";
+import { renderTaskPageComponent } from "@/pages/tasks/[taskUrlSlug]";
```

Comment:
[dust] I generally don't like this pattern. If there's functionality you think should be brought in from a route handler, then chances are it needs to be abstracted and separated into its own file, then imported here and in the route handler.

Reply:
done